doc/developer-notes.md

```diff
@@ -249,6 +250,23 @@ Development tips and tricks
 Run configure with `--enable-debug` to add additional compiler flags that
 produce better debugging builds.
 
+### Show sources in debugging
+
+Absolute paths are stripped from debug information with -fdebug-prefix-map and
+-fmacro-prefix-map options. In case you move binaries after compilation or use
```

Comment:
May be worth noting some or all of:

 * this is only when compiling with ccache enabled (and only if the compiler supports it)
 * that it breaks running "lldb ./bitcoind" (etc) when your cwd is the src/ directory, even if you don't move the file around (gdb magically works fine, it seems) -- and that this can be worked around by creating a "src -> ." symlink within the src/ directory.
 * the debugedit binary (available on debian/red hat based distros) can be used to edit the binaries to make the debug path absolute as well

Reply:
Updated. Please take another look